main.go

```diff
@@ -154,6 +154,7 @@ func main() {
 			Scheme:        mgr.GetScheme(),
 			ClientCache:   clientCache,
 			SeedsResolver: cassandra.NewRemoteSeedsResolver(),
+			ManagementApi: cassandra.NewManagementApiFacade(),
```

Comment:
I am not a fan of having the `ManagementApi` field in the Reconciler. Every method that we add to the `ManagementApiFacade` interface will have to include connection info. More importantly, we have to initialize the connection on each method invocation. That is overhead we could otherwise avoid. I will create a follow up issue for this.